Elecritic/Pages/ProductPage.razor.cs

```diff
@@ -48,13 +95,9 @@ public partial class ProductPage {
                 Product = Product
             };
 
-            bool publicationSucceeded = await ProductContext.InsertReviewAsync(review);
-            if (publicationSucceeded) {
-                PublicationMessage = "¡Reseña publicada con éxito!";
-            }
-            else {
-                PublicationMessage = "Lo sentimos, tu reseña no pudo ser publicada :(";
-            }
+            var publicationSucceeded = await ProductContext.InsertReviewAsync(review);
+            PublicationMessage = publicationSucceeded ?
+                "¡Reseña publicada con éxito!" : "Lo sentimos, tu reseña no pudo ser publicada :(";
```

Comment:
C# wizardry I still don´t understand fully, but Im guessing it works as the if else statement

Reply:
> Im guessing it works as the if else statement

indeed it does, it's the ternary conditional operator, see https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/conditional-operator

Reply:
> > Im guessing it works as the if else statement
> 
> indeed it does, it's the ternary conditional operator, see https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/conditional-operator

Nice, thank you, I liked the mnemonic device to remember it: is this condition true ? yes : no